Modelica/Blocks/Sources.mo

```diff
@@ -598,7 +598,7 @@ and that the parameter <code>startTime</code> is omitted since the voltage can b
     SI.Angle x=2*pi*f*(time - startTime);
   equation
     y = offset + (if time < startTime then 0 else amplitude*
-      (if noEvent(time - startTime)<eps then 1 else (sin(x))/x));
+      (if noEvent(time - startTime < eps) then 1 else (sin(x))/x));
```

Comment:
Wouldn't it be numerically better to use `x` in the comparison, as this should avoid the risk of the condition being transformed into `time < startTime + eps` where the addition with `eps` could suffer from truncation unless `startTime` is close to zero?

Reply:
It would be more consistent, even though I would have to check if such changes are allowed.
It can also be combined with using `<=` instead of `<`, as that causes any truncation result to take the then-branch.